bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -425,6 +307,15 @@ impl fmt::Display for LockTime {
     }
 }
 
+impl FromHexStr for LockTime {
+    type Error = Error;
+
+    fn from_hex_str_no_prefix<S: AsRef<str> + Into<String>>(s: S) -> Result<Self, Self::Error> {
+        let packed_lock_time = crate::parse::hex_u32(s)?;
+        Ok(Self::from_consensus(packed_lock_time))
```

Comment:
Perhaps this local variable could be better name, `packed_lock_time` is an artifact of `PackedLockTime`, right?

Reply:
No, I think it's fine ... we are parsing a `u32`, which can reasonably be described as a "packed locktime" then we're unpacking it into an enum type.